utils/github_utils.py

```diff
@@ -0,0 +1,11 @@
+import requests
+
+from github import Github
+from flask import current_app as app
+from constants import MASADER_GH_REPO
+
+def report_issue(title: str, msg:str):
+    git_con = Github(app.config['GH_USERNAME'], app.config['GH_PASSWORD'])
```

Comment:
Can'r we use access token instead of username and password? From the package GitHub readme I can see this `g = Github("access_token")`.